source/product-stories/chargy.html.erb

```diff
@@ -0,0 +1,154 @@
+---
+title: "Product Story - char.gy - Providing residents with an on-street electric vehicle lamppost charging solution with char.gy"
+meta_keywords: "Ruby on Rails, Ruby on Rails developer, Ruby on Rails developers, Ruby on Rails development, Ruby on Rails developer London"
```

Comment:
I don't think any of these are relevant for this product story. I'd leave them out.